Bio/PDB/ResidueDepth.py

```diff
@@ -448,8 +446,7 @@ def _read_vertex_array(filename):
 
 
 def get_surface(model, PDB_TO_XYZR=None, MSMS="msms"):
-    """
-    Return a Numpy array that represents
+    """Return a Numpy array that represents
```

Comment:
Leave the main text as it was, insert a one line summary like ``Represent molecular surface as a vertex list array.``